force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,7 @@
+trigger OpportunityTrigger on Opportunity (before update) {
+    if(trigger.isUpdate){
+        if(trigger.IsBefore){
+            OpportunityManager.beforeUpdateOpportunity(Trigger.New,Trigger.oldMap);
```

Comment:
Since this trigger only works with `before update`, so don't need to double-check that condition again